### DISCUSSION TITLE
Enable sf org create user for sandboxes

### DISCUSSION THREAD

**Discussion by pkozuchowski** [forcedotcom/cli#2606] [Ideas]
I'd like to use `sf org create user` with user definition file to quickly create test users on sandboxes, but the command fails with "auto-approving is only allowed in scratch orgs".

My use-cases are:
- On QA / UAT / Preprod environments I will have to create many users, for different people, with the same Profile & Permission sets for testing purposes. That is for testers, BAs, UAT users, developers.
- Setting up Integration users quickly on many sandboxes in the pipeline.

These users do not need to be available through CLI.

**Comment by mshanemc** [comment]
it sounds kinda like
1. turn config file into a user in the org (like normal, with profile/permssets)
2. but without auth.   Like maybe just a `--no-auth` option?
